.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,29 @@
+# Contributing to tidymodels
+
+For more detailed information about contributing to tidymodels packages, see our [**development contributing guide**](https://www.tidymodels.org/contribute/).
+
+## Documentation
+
+Typos or grammatical errors in documentation may be edited directly using the GitHub web interface, as long as the changes are made in the _source_ file.
+
+*  YES ✅: you edit a roxygen comment in an `.R` file in the `R/` directory.
+*  NO 🚫: you edit an `.Rd` file in the `man/` directory.
+
+We use [roxygen2](https://cran.r-project.org/package=roxygen2), with [Markdown syntax](https://cran.r-project.org/web/packages/roxygen2/vignettes/rd-formatting.html), for documentation.
+
+## Code
+
+Before you submit 🎯 a pull request on a tidymodels package, always file an issue and confirm the tidymodels team agrees with your idea and is happy with your basic proposal.
+
+The [tidymodels packages](https://www.tidymodels.org/packages/) work together. Each package contains its own unit tests, while integration tests and other tests using all the packages are contained in [extratests](https://github.com/tidymodels/extratests).
+
+*  We recommend that you create a Git branch for each pull request (PR).
```

Comment:
```
* For pull requests, we recommend that you create a fork of this repo with `usethis::create_from_github()`, and then initiate a new branch with `usethis::pr_init()`.
```

It might be useful to mention some usethis functions here

I'll merge this anyways for now and we can revisit later if that seems useful for newcomers